website/docs/r/api_management_api_policy.html.markdown

```diff
@@ -46,7 +46,7 @@ The following arguments are supported:
 
 * `resource_group_name` - (Required) The name of the Resource Group in which the API Management Service exists. Changing this forces a new resource to be created.
 
-* `xml_content` - (Optional) The XML Content for this Policy.
+* `xml_content` - (Optional) The XML Content for this Policy as a string. Note instead of parameter for Microsoft's PolicyFilePath. xml-content can take a file by using Terraform's [file function](https://www.terraform.io/docs/configuration/functions/file.html)
```

Comment:
I think this could sound better as:
```suggestion
* `xml_content` - (Optional) The XML Content for this Policy as a string. An XML file can be used here with Terraform's [file function](https://www.terraform.io/docs/configuration/functions/file.html) that is similar to Microsoft's `PolicyFilePath` option.
```
WDYT?